index-handlers/src/test/java/no/sikt/nva/nvi/index/OpenSearchClientTest.java

```diff
@@ -425,6 +462,13 @@ private static CandidateSearchParameters.Builder defaultSearchParameters() {
                    .withCustomer(CUSTOMER).withUsername(USERNAME).withYear(YEAR);
     }
 
+    private static String getRandomWord(String str) {
```

Comment:
You can replace this with static method `RandomDataGenerator.randomString()` from (no.unit.nva.testutils)